docs/source/developing/releasing.adoc

```diff
@@ -12,15 +12,23 @@ The following sections describe how to release Minishift.
 [[release-prereqs]]
 == Prerequisites
 
-You must have a https://help.github.com/articles/creating-an-access-token-for-command-line-use[GitHub personal access token]
+* You must have a https://help.github.com/articles/creating-an-access-token-for-command-line-use[GitHub personal access token]
 defined in your environment as _GITHUB_ACCESS_TOKEN_.
+* You need _gnutar_ installed on your machine. See also GitHub issue link:https://github.com/minishift/minishift/issues/657[#657]
+
+== Preparing the GitHub milestone
+
+. Verify the milestone you want to release
+ * Move non resolved issues into the next milestone
+ * Review resolved/closed issues and make sure they are classified correctly
+ * Check the resolution of the issues. For example, if some code has changed as part of
+   the milestone, the resolution should most likley be _resolution/done_.
```

Comment:
No full stop at the end of line 24 or have full stop in above two points for consistency.